### ISSUE TITLE
Update to latest UI Shell for a11y fixes

### ISSUE THREAD

**Issue by alisonjoseph** [IBM-Design/design-website#303]
blocked by: https://github.com/IBM/carbon-components-react/pull/1813#pullrequestreview-199162372

Once PR above is merged in we can bring in latest Shell component to fix keyboard a11y issues with navigation.

**Comment by dakahn** [comment]
Fixed! PR was merged into carbon-components-react
